hscontrol/app.go

```diff
@@ -761,25 +768,9 @@ func (h *Headscale) Serve() error {
 					Msg("Received SIGHUP, reloading ACL and Config")
 
 				// TODO(kradalby): Reload config on SIGHUP
```

Comment:
```suggestion
```

Reply:
This still isn't implemented. I'll keep this for the time being.

Reply:
Should probably not allow "large" TODOs like this without an issue... (including me)